package.json

```
@@ -41,22 +41,19 @@
         "@commitlint/cli": "^8.3.5",
         "@commitlint/config-conventional": "^8.3.4",
         "@escommunity/minami": "^1.0.0",
-        "@slimio/eslint-config": "^4.1.0",
-        "@slimio/psp": "^0.11.1",
+        "@nodesecure/eslint-config": "^1.7.0",
         "codecov": "^3.7.0",
-        "cross-env": "^7.0.2",
-        "eslint": "^7.0.0",
+        "eslint": "^8.32.0",
         "husky": "^4.2.5",
-        "jest": "^26.0.1",
         "jsdoc": "^3.6.4",
         "pkg-ok": "^2.3.1"
```

```suggestion
        "pkg-ok": "^3.0.0"
```

Github doesn't allow me to write only one comment for all dependencies 😬 The goal is to update them all.
I'm not sure if we should remove husky, commitlint. I've also seen @fraxken removing `pkg-ok` in other package, right ?